pkg/web/api/v1/user.go

```diff
@@ -123,3 +133,20 @@ func (u UserPassword) Validate() (err error) {
 
 	return err
 }
+
+//===========================================================================
+// User Query
+//===========================================================================
+
+func (q *UserQuery) Validate() (err error) {
+	q.Detail = strings.ToLower(strings.TrimSpace(q.Detail))
+	if q.Detail == "" {
+		q.Detail = DetailUser
```

Comment:
Although we aren't currently displaying a user detail modal, it made sense to make that the default going forward.